src/app/collective-rewards/rewards/hooks/useGetChartRewardsData.tsx

```diff
@@ -0,0 +1,27 @@
+import { AVERAGE_BLOCKTIME } from '@/lib/constants'
+import { useQuery } from '@tanstack/react-query'
+import { CycleRewardsItem } from '@/app/collective-rewards/types'
+
+export const useGetChartRewardsData = () => {
+  const { data, isLoading, error } = useQuery<CycleRewardsItem[], Error>({
+    queryFn: async (): Promise<CycleRewardsItem[]> => {
+      const response = await fetch(`/api/cycles`)
```

Comment:
same as for the other api, please specify the pageSize. Ideally, the hooks should receive the page size as parameter and we should pass it to the fetch